static/js/user_groups.ts

```diff
@@ -8,6 +8,7 @@ type UserGroup = {
     name: string;
     members: Set<number>;
     is_system_group: boolean;
+    subgroups: Set<number>;
```

Comment:
Do we want to call this `direct_subgroups` for clarity? It seems like it might be helpful given that some functions will want to look at the full recursively computed set of subgroups of groups.

Reply:
Yes, we can do that. In that case do we want same in the user group objects sent from server too?

Reply:
It would be better to do change in the backend at this point of time as clients do not use it currently, so no issue of backwards compatibility and I observed that would also result in duplicate code in tests if server returns `subgroups` field and we use `direct_subgroups` here.